support-frontend/assets/components/marketingConsent/marketingConsentPaper.jsx

```diff
@@ -0,0 +1,187 @@
+  ${headline.xxsmall({ fontWeight: 'bold', lineHeight: 'tight' })};
+  ${from.desktop} {
+    line-height: normal;
+  }
+`;
+
+const sansText = css`
+  ${textSans.medium({ lineHeight: 'regular' })}
+  ${from.desktop} {
+    ${textSans.medium({ lineHeight: 'loose' })}
+  }
+`;
+
+const tinyText = css`
+  ${textSans.small({ fontStyle: 'italic' })};
+  margin-bottom: ${space[5]}px;
+`;
+
+const maxWidth = css`
+  ${from.tablet} {
+    max-width: 70%;
+  }
+
+  ${from.leftCol} {
+    max-width: 60%;
+  }
+`;
+
+const marginForButton = css`
+  margin: ${space[5]}px 0 0;
+`;
+
+// ----- Types ----- //
+
+type ButtonPropTypes = {|
+  confirmOptIn: ?boolean,
+  email: string,
+  csrf: CsrfState,
+  onClick: (?string, CsrfState) => void,
+  requestPending: boolean,
+|};
+
+type PropTypes = {|
+  ...ButtonPropTypes,
+  error: boolean,
+|};
+
+const mapStateToProps = state => ({
+  confirmOptIn: state.page.marketingConsent.confirmOptIn,
+  email: getEmail(state),
+  csrf: state.page.csrf,
+});
+
+function mapDispatchToProps(dispatch: Dispatch<Action>) {
+  return {
+    onClick: (email: string, csrf: CsrfState) => {
+      sendTrackingEventsOnClick({
+        id: 'marketing-permissions',
+        componentType: 'ACQUISITIONS_OTHER',
+      })();
+      sendMarketingPreferencesToIdentity(
+        true, // it's TRUE because the button says Sign Me Up!
+        email,
+        dispatch,
+        csrf,
+        'MARKETING_CONSENT',
+      );
+    },
+  };
+}
+
+
+// ----- Render ----- //
+
+function MarketingButton(props: ButtonPropTypes) {
+  const confirmedOrPending = props.confirmOptIn === true || props.requestPending === true;
+  const confirmedOptIn = props.confirmOptIn === true;
+  if (confirmedOrPending) {
+    return (
+      <ThemeProvider theme={buttonBrandAlt}>
+        <Button
+          css={marginForButton}
+          iconSide="right"
+          icon={<SvgEnvelope />}
+          disabled
+        >
+          {confirmedOptIn ? 'Signed up' : 'Pending...'}
+        </Button>
+      </ThemeProvider>
+    );
+  }
+
+  return (
+    <ThemeProvider theme={buttonBrandAlt}>
+      <Button
+        css={marginForButton}
+        iconSide="right"
+        aria-label="Sign me up to news and offers from The Guardian"
+        onClick={() => props.onClick(props.email, props.csrf)}
+        icon={<SvgEnvelope />}
+      >
+        Sign me up
+      </Button>
+    </ThemeProvider>
+  );
+}
+
+function MarketingConsent(props: PropTypes) {
+
+  if (props.error) {
+    return (<GeneralErrorMessage
+      classModifiers={['marketing_consent_api_error']}
+      errorHeading="Sorry, something went wrong"
+      errorReason="marketing_consent_api_error"
+    />);
+  }
+
+  if (checkEmail(props.email)) {
+    return (
+      <section css={maxWidth}>
+        <h3 css={subHeading}>Contributions, subscriptions and membership</h3>
+        <p css={sansText}>
+          Get related news and offers &ndash; whether you are a contributor, subscriber, member or
```

Comment:
I know this text is common to a number of pages but the 'whether [you] would like to become one' line seems completely inappropriate to a thank you page!